docs/source/authentication.rst

```diff
@@ -33,6 +33,13 @@ Or the same example using ``curl``:
 
     curl -u username:password -H 'Accept:application/json' $URL
 
+.. warning::
```

Comment:
I would move that warning into the JSON web token section as it only affects JWT.